### DISCUSSION TITLE
Serious problems and recommendations up to version 1.2.7

### DISCUSSION THREAD

**Discussion by JeRsVe** [Chia-Network/chia-blockchain#8574] [Support]
Greetings.
I am aware that any new project will always have more problems at the beginning, but I also know as a programmer that the effort you have made is enormous.
I congratulate you for everything you have achieved so far and how you have managed to overcome part of the problems included with the creation of a new product such as chia coins.
I have had a wide variety of problems and I have some observations that I know will help us all to improve the development and manipulation of farms and funds.

My main problem has been with the database that keeps the synchronization that due to a long drop in electricity service for more than 6 hours it did not synchronize again and there has been no way for it to continue growing, so I infer that there is a problem with the blockchain_v1_mainnet.sqlite file and there should be some maintenance routine to recover or repair said file, without the days it takes to re-synchronize, on the other hand thinking that the file would be launched with the restart of the computer and installing Windows 10 x64 again and starting all over again and not including but my 24 input words.

Well wiht the 1.2.7 it doesn't work either because it stays in synchronization at 225,760 and from there it does not advance for more than 4 days trying to wait for it to solve the problem, which the program does not do either, which should have a counter included on the synchronization time and detect the inactivity and the possible reasons and even solutions, I don't think it will be very difficult for you to do.

I tried to skip the problem by copying again where I was reduced to 816,200 and I already have 5 days lost again, for a total of one month and two weeks with 3 days for the total problems that I have had to overcome, but the frustrating thing is that when you finally have everything walking and you have already achieved with a lot of sacrifice and money to have 0.002525807533 XCH then everything stops and there is no way to recover what was walking so well. 

On the other hand, the control of the pools should be broader for the user, from the total elimination of any of those that appear in the window such as the rearm of the pools present with the farms that are already working without problems, because when it starts Again the program from scratch as it turns out that there is no record on your part of which farms are related to which pools, nor how much is counted in the wallet and much less if the synchronization process has any problem, I understand that it is a hard work but there are things that are logical that could be done, simply be the users of your product and try and force to the maximum the situations that put the stability of the program at risk and even worse that the clients are sure that you have a copy of what has been done in each associated account, I am not saying the details, but at least the final results, that is, where it is synchronized, how much it has in its wallet, what can ols is using and which farms are affiliated, I think that if it takes forever to allow synchronization it is well worth it that they also have the information of each account for their restart in emergency chaos, I know that this would bring many more clients because it would give them more security on your investment in the chia coin project.

Please I need your help because in reality I do not know what else to invent so as not to waste more time and time is money.

My name is Jorge Romero and I live in Venezuela in South America and if you do not know what country this is, please take a look and you will understand with what sacrifice I am doing all this.

May God bless you in everything you are doing and may you open your minds and find the necessary solutions so that your project is taken more seriously.

I await your prompt response, not an automatic one but a written one with the necessary knowledge to improve our chia coin.

My email: jersve@gmail.com
Working on windows 10 x64
Intel I5 2nd Generation
8 Gb Ram
HDD 8 Tb

**Comment by loppefaaret** [comment]
Perhaps you can find something in your `debug.log` file, located under your users folder `~\.chia\mainnet\log\` that can tell us what is going on with the blockchain database, what is preventing your node from syncing ?

**Comment by BrandtH22** [comment]
@JeRsVe , the newest version of chia has fixes in place to resolve this issue including a new database compression and the abiltiy to download a snapshot of the database. Please download the latest version of chia and the database snapshot from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket
